packages/components/src/card/card/hook.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { cx } from 'emotion';
+
+/**
+ * WordPress dependencies
+ */
+import deprecated from '@wordpress/deprecated';
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useContextSystem } from '../../ui/context';
+import { useSurface } from '../../surface';
+import * as styles from '../styles';
+
+/**
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../types').Props, 'div'>} props
+ */
+function useDeprecatedProps( { elevation, isElevated, ...otherProps } ) {
+	let computedElevation = elevation;
+
+	if ( isElevated ) {
+		deprecated( 'Card isElevated prop', {
+			since: '5.9',
+			alternative: 'elevation',
+		} );
+		computedElevation ??= 2;
+	}
+
+	return {
+		...otherProps,
+		isElevated,
```

Comment:
Why do we continue to return `isElevated` here? Shouldn't we just return `elevation`?

Reply:
That's my bad, I meant to delete that line but it escaped my refactor.

Done in https://github.com/WordPress/gutenberg/pull/32566/commits/fe42e0d2ff67979ced5200c279ccba5e5de67344